pkg/cmd/pulumi/gen_markdown.go

```diff
@@ -31,6 +31,9 @@ import (
 // Used to replace the `## <command>` line in generated markdown files.
 var replaceH2Pattern = regexp.MustCompile(`(?m)^## .*$`)
 
+// Used to promote the `###` headings to `##` in generated markdown files.
+var h3Pattern = regexp.MustCompile(`(?m)^###`)
```

Comment:
Shouldn't this have a space (like the H2 pattern above it), so it doesn't match anything with more than three consecutive `#s`?